duckduckgo/duckduckgo-privacy-extension
# HTTPS: don't strip out www from domain

**Reviewer:** @jdorweiler 


**CC:** @zachthompson 


## Description:
We currently strip out the "www" from domains when we check if they're in the list or not. 
Since some domains in the list contain the "www" part, we should not do that.


## Steps to test this PR:

1. Build and reload extension
2. Try loading some of the  domains with "www" in the list (eg "mozdev.org", "nowmentalhealth.com", "bosch-home.com.au", "banche.roma.it") 
3. Open the popup. Expected: the https line should say "Encrypted Connection" with a green lock icon
4. Also try some sites without the "www"
5. Repeat step 3, with same expected result

I've also added some unit tests for utils.es6.js

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
